static/app/views/settings/projectDebugFiles/sources/customRepositories/index.tsx

```diff
@@ -175,6 +176,21 @@ function CustomRepositories({
     });
   }
 
+  async function handleSyncRepositoryNow(repoId: CustomRepo['id']) {
+    try {
+      await api.requestPromise(
+        `/projects/${orgSlug}/${projSlug}/appstoreconnect/${repoId}/refresh/`,
+        {
+          method: 'POST',
+        }
+      );
+    } catch (error) {
+      const errorMessage = t('Unable to refresh repository. Try again in a few minutes.');
```

Comment:
I find the "unable to fresh repository" part too vague: it does not say this is rate limited.  Maybe "Rate limit for refreshing repository exceeded. Try again in a few minutes." is better?

Reply:
sure thing. Great suggestion! thanks 😉